spyder/plugins/editor/confpage.py

```diff
@@ -50,29 +50,40 @@ def setup_page(self):
         # ---- Display tab
         showtabbar_box = newcb(_("Show tab bar"), 'show_tab_bar')
         show_filename_box = newcb(
-            _("Show full file name on top of tab bar"),
+            _("Show full file path above editor"),
             'show_filename_toolbar'
         )
         showclassfuncdropdown_box = newcb(
-                _("Show selector for classes and functions"),
+                _("Show class/function selector"),
```

Comment:
I think the previous text is easier to translate to other languages. What's wrong about it?

Reply:
This is the same wording change we agreed on for the `Source` menu, so its just changed it to be consistent with the menu. It reads clearer and more idiomatic to me like this in English, but as the meaning is identical, it need not be translated differently than before if the existing translation makes sense and sounds idiomatic in that language.